.pipelines/Package-Official.yml

```diff
@@ -52,8 +52,7 @@ extends:
         Network: KS3
       WindowsHostVersion:
         Version: 2022
-        # Azure container/blob operations get blocked when using KS3
-        Network: KS2
+        Network: KS3
```

Comment:
really should be R1 but can be in a future PR

Reply:
Yes, will change to use R1 in a separate PR. Opened #367 to track.